discord/guild.go

```diff
@@ -228,14 +323,16 @@ func (guild *GuildState) handleMessageCreate(s *discordgo.Session, m *discordgo.
 						//TODO print usage of this command specifically
 						s.ChannelMessageSend(m.ChannelID, "You used this command incorrectly! Please refer to `.au help` for proper command usage")
 					} else {
-						channelName := strings.Join(args[1:], " ")
+						// if anything is given in the second slot then we consider that a true
+						forGhosts := len(args[2:]) >= 1
```

Comment:
cheeky way of checking for any value, would need to be changed if we want to add more params to the command similarly to `channelName`